Tests/SuperLinq.Test/FallbackIfEmptyTest.cs

```diff
@@ -10,6 +10,22 @@ public void FallbackIfEmptyWithEmptySequence()
 		source.FallbackIfEmpty(12, 23).AssertSequenceEqual(12, 23);
 	}
 
+	[Fact]
+	public void FallbackIfEmptyWithEmptyListSequence()
```

Comment:
Test is unnecessary; version with `AsTestingCollection()` is appropriate to cover.